cmd/kubectl-gadget/deploy.go

```diff
@@ -183,6 +184,10 @@ func init() {
 		&daemonLogLevel,
 		"daemon-log-level", "", "info", fmt.Sprintf("Set the ig-k8s log level, valid values are: %v", strings.Join(strLevels, ", ")))
 	rootCmd.AddCommand(deployCmd)
+	deployCmd.PersistentFlags().StringVarP(
+		&appArmorprofile,
+		"apparmor-profile", "", "unconfined", "AppArmor profile to use, unconfined by default")
```

Comment:
Remove `, unconfined by default`, otherwise this information will be kind of duplicated: `AppArmor profile to use, unconfined by default (default "unconfined")`